src/__tests__/utils.test.ts

```diff
@@ -3,7 +3,7 @@ import { expect } from "chai";
 
 import { AssetTransfer } from "../parser/csvParser";
 import { testData } from "../test/util";
-import { fromWei, toWei, TEN, ONE, ZERO, transfersToSummary } from "../utils";
+import { fromWei, toWei, TEN, ONE, ZERO, assetTransfersToSummary } from "../utils";
```

Comment:
Now there are two summary functions:

- assetTransfersToSummary
- collectibleTransfersToSummary

thats why I renamed this one more specific.
But the test could also just skip testing this small function because we can actually unit test the balance check now.